### PR TITLE
Featuer/issue 152 privacy policy page

### PR DESCRIPTION
**Pull request by ricjon**

### CODE REVIEW

File: client/templates/nav/header.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
           <li><a href="{{pathFor 'about'}}">{{_ "about"}} <span class="sr-only">(current)</span></a></li>
           <li><a href="{{pathFor 'faq'}}">{{_ "faq"}}</a></li>
           <li><a href="{{pathFor 'browse'}}">{{_ "browse"}} <span class="sr-only">(current)</span></a></li>
+          <li><a href="{{pathFor 'privacy'}}">{{_ "privacy"}} <span class="sr-only">(current)</span></a></li>
```
Comment (ricjon) [inline client/templates/nav/header.html]: Moved it to footer.

File: client/templates/other/privacy.html [hunk 2]
```diff
@@ -1,24 +1,182 @@
 <template name="privacyPolicy">
-<div id="privacy">
-    <div class="section privacy-top">
-		  <div class="container">
-		    <div class="row">
-		        <div class="col-md-12 align-center">
-		          <h2>Codebuddies Privacy Policy</h2>
```
Comment (wuworkshop) [inline client/templates/other/privacy.html]: For consistency, could you replace "Codebuddies" on line 7 with "CodeBuddies"?

Comment (ricjon) [inline client/templates/other/privacy.html]: It's fixed now.

**Comment by lpatmo** [comment]
Thank you @ricjon! :)

**Pull request merged**